### ISSUE TITLE
Watching long videos at 1.5x sometimes sends video back to beginning instead of skipping sponsor.

### ISSUE THREAD

**Issue by K4sum1** [ajayyy/SponsorBlock#1889]
For most videos in a live stream or podcast format, I watch them at 1.5x. I've found for some videos that instead of skipping the sponsors, I get sent back to the beginning. This issue appears to come and go, so I'm not sure how repeatable it is, but I had it happen for this video.

https://www.youtube.com/watch?v=aaiP7cQPwck

**Comment by ajayyy** [comment]
Firefox? Segments at the end or other times?

**Comment by K4sum1** [comment]
Yeah Firefox, specifically Developer Edition. The video I linked has two segments, both a good amount in the video. It's usually only these long Broken Silicon videos where I have had this happen. It happened for both of them in this video for me, and it happened for both in the previous video I think.

**Comment by ajayyy** [comment]
But is it segments in the middle where it happens, or ones that skip to the end

**Comment by K4sum1** [comment]
There's nothing at the end. If I reach the segments in the middle, I get sent back to the beginning.

**Comment by ajayyy** [comment]
What may be happening is, when the skip happens, it causes it to change video resolutions due to taking too long to buffer (which causes the video time to be reset to zero). Normally, the video player gets reset to the correct position and it fixes itself, but maybe somehow the segment at the beginning auto skipping early enough is causing progress in the video to be lost. Would explain why it is only on 1.5x speed and with long videos.

**Comment by BaconCatBug** [comment]
This is happening to me as well now. 2x speed, on firefox, with a mid-video segment. Instead of skipping to the end of the midroll segment, it resets back to the start.

**Comment by AsheIsAmazing** [comment]
Can confirm the same issue, also on firefox on videos with a beginning segment.

**Comment by j-bs** [comment]
Another confirmation of mid-video skip on firefox resetting the video here too. Watching 2x

**Comment by 4fosaefp** [comment]
I'm experience the same issue on Firefox. Happens occasionally even on shorter videos (~10min). I also watch at 1.5-2x speed.
When there's a sponsored segment at end of the video, it'll jump back to the start.

**Comment by Anizam** [comment]
Same problem with Firefox

```
Version 122.0 (64-bit)
Mozilla Firefox Snap for Ubuntu
canonical-002 - 1.0
```
Plugin

```
5.5.3
```

**Comment by D0han** [comment]
Same, Firefox on Linux, usually 1.5-3.0 speed

**Comment by 0ctoqus** [comment]
Same problem here in Firefox and windows when watching at 2x speed

**Comment by Chalenor** [comment]
Ditto. Watching this video at 1.5x in Firefox's default stable build in Windows 10 on 1 Gbps connection, and it can randomly occur during a segment skip: https://www.youtube.com/watch?v=K4oPpnAEvz8

**Comment by ajayyy** [comment]
If anyone is able to catch a recording,  that would help. Knowing timing of how quick it happens after the skip would make it possible for the extension to detect this and automatically seek back to where it was

**Comment by Chalenor** [comment]
Got a video of the event here:

https://github.com/ajayyy/SponsorBlock/assets/71306819/ca159bed-b5d0-4ae2-b394-a80701bc6404

**Comment by ajayyy** [comment]
thanks! Does look like it is changing resolutions

**Comment by vladchirila** [comment]
It recently started happening for me as well.
I use Firefox on Windows, and normally watch videos at 1.5x speed.
It just happened for me for this video: https://www.youtube.com/watch?v=1MX6ZK8VPto
When it reaches the green segment around 22 minutes, it skips straight to the beginning.
